model/export/ColumnsProvider.php

```diff
@@ -212,6 +216,25 @@ public function getResponseColumns()
         return $this->resultsService->getVariableColumns($this->delivery, \taoResultServer_models_classes_ResponseVariable::class);
     }
 
+    /**
+     * Returns all trace variables columns to be exported.
+     *
+     * @throws \RuntimeException
+     * @return array
+     */
+    public function getTraceVariablesColumns()
+    {
+        return [
+            [
+                'type' => TraceVariableColumn::class,
+                'label' => self::LABEL_TRACE_VARIABLES,
+                'contextId' => 'delivery_execution',
```

Comment:
can we retrieve this and the value 1 line below by calling TraceVariableColumn->getContextIdentifier() ? so that we don't duplicate this